### PR TITLE
Boilerplate Spec for master Branch

### PR DESCRIPTION
**Pull request by dblevins**
Generated boilerplate spec.  Feel free to merge, then refine.

 See https://wiki.eclipse.org/How_to_Prepare_API_Projects_to_Jakarta_EE_8_Release

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,18 +17,29 @@
...
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.4</version>
```
Comment (kwsutter) [inline pom.xml]: Shouldn't this be 1.0.5?

File: spec/src/main/asciidoc/management-spec.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2017, 2019 Contributors to the Eclipse Foundation
+//
+
+= Jakarta Management 1.1
+:authors: Jakarta Management Team, https://projects.eclipse.org/projects/ee4j.jakartaee-stable
+:email: https://dev.eclipse.org/mailman/listinfo/management-api-dev
```
Comment (kwsutter) [inline spec/src/main/asciidoc/management-spec.adoc]: mailing list is jakartaee-stable-dev

**Comment by kwsutter** [comment]
Per conversation on mailing list, I'm going to just merge this PR and then do a separate PR to do these minor updates.

**Comment by dblevins** [comment]
This is broken -- I'll fix


-- 
David Blevins
http://twitter.com/dblevins
http://www.tomitribe.com

**Pull request merged**